lib/phoenix/live_dashboard/live/metrics_live.ex

```diff
@@ -70,25 +70,42 @@ defmodule Phoenix.LiveDashboard.MetricsLive do
     <%= if @metrics do %>
       <div class="phx-dashboard-metrics-grid row">
       <%= for {metric, id} <- @metrics do %>
-        <%= live_component @socket, ChartComponent, id: id, metric: metric %>
+        <%= live_component @socket, ChartComponent, id: id, metric: metric,
+          do: metric
+              |> history_for(id, @historical_data)
+              |> send_updates_for_entries()
```

Comment:
Instead of sending those as updates, they should be initial data given to the component.

Reply:
Hmm, I tried this initially and it didn't work, I can see if that changed or if I can figure out how to rework, but I think in the JS code the assumption that it starts with no data is fairly strong... I'll start looking into though, and/or if anyone else has any guidance on how to do this properly input is welcome.

Reply:
I see. Doing it here on the `do` block is not appropriate though. We can most likely do it on `mount`.

Reply:
I'll look into trying that way, I don't have a ton of experience with LiveView but hopefully good opportunity to learn.

Reply:
Apologies, just to clarify, did you have in mind doing this in `MetricsLive.mount` or in `ChartComponent.mount`?  I'd previously thought to do in `ChartComponent.mount` and couldn't figure out how or if to access the context needed there, so I'm guessing you mean `MetricsLive.mount` but thought worth double checking in case I'm confused/there's a way I don't understand from lack of LiveView experience.

Reply:
In MetricsLive.